build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -317,8 +317,8 @@ var baseUrl = (function () {
   // base href should start with "/"
   if (typeof document !== 'undefined') {
     var el = document.getElementsByTagName('base');
-    if (el && el[0] && el[0].href && el[0].href.startsWith('/')){
-    	return el[0].href;
+    if (el && el[0] && el[0].href && el[0].getAttribute("href") && el[0].getAttribute("href").startsWith("/")){
```

Comment:
I'm confused why we need both `el[0].href` and `el[0].getAttribute("href")`

Should the first one be removed?